mmv1/third_party/terraform/acctest/tgc_utils.go

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
These comments will be added automatically.

Reply:
removed